app/views/podcast_episodes/_meta.html.erb

```diff
@@ -24,13 +24,13 @@
 <% else %>
   <% title "Podcasts" %>
   <%= content_for :page_meta do %>
-    <link rel="canonical" href="https://<%= ApplicationConfig["APP_DOMAIN"] %>/pod" />
+    <link rel="canonical" href="<%= app_url("/pod") %>" />
```

Comment:
Perhaps we can use `pod_path` here instead of `"/pod"`, I know we've been using the stringed url everywhere else but since the mailers PR trying to use the routes url_helpers where we can